src/sql/src/plan/query.rs

```diff
@@ -6124,7 +6131,7 @@ impl Visit<'_, Aug> for WindowFuncCollector {
     }
 
     fn visit_query(&mut self, _query: &Query<Aug>) {
-        // Don't go into subqueries. Those will be handled by their own `plan_view_select`.
+        // Don't go into subqueries. Those will be handled by their own `plan_query`.
```

Comment:
Just confirming it's not `plan_select_from_where`.

Reply:
Yep, `plan_subquery`/`plan_nested_query` calls `plan_query`. Even though `plan_query` does eventually call `plan_select_from_where`, it would be a little misleading to say that a subquery has its own `plan_select_from_where`, as if that were the whole subquery.